packages/ts/file-router/src/runtime.ts

```diff
@@ -1,4 +1,4 @@
 export * from './runtime/createRoute.js';
-export * from './runtime/toReactRouter.js';
+export * from './runtime/RouterConfigurationBuilder.js';
```

Comment:
IT tests are failing due to missing `toReactRouter` export, which is used in Flow. Is there a way to keep it for now, and then remove after we update Flow?

Reply:
let me try a suggestion:
```suggestion
export * from './runtime/toReactRouter.js';
export * from './runtime/RouterConfigurationBuilder.js';
```